app/react/src/client/preview/render.js

```diff
@@ -3,14 +3,13 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
 import { stripIndents } from 'common-tags';
+import logger from 'npmlog';
```

Comment:
Does it work on client?

Reply:
(facepalm) It was late when I was working on this. Thanks @Hypnosphi